gui/src/installer/view.rs

```diff
@@ -392,7 +393,11 @@ pub fn import_descriptor<'a>(
             form::Form::new_trimmed("Descriptor", imported_descriptor, |msg| {
                 Message::DefineDescriptor(message::DefineDescriptor::ImportDescriptor(msg))
             })
-            .warning("Incompatible descriptor.")
+            .warning(if wrong_network {
+                "The descriptor is for an other network, you can change the network in the previous steps."
+            } else {
+                "Incompatible descriptor."
```

Comment:
Could we have a less coloured error message in the general case? Like "error reading descriptor". "Incompatible" can freak a user out as it can be read as "we successfully read your descriptor and it's not compatible with this software", and it's often not the case.